static/resources/yaml/observability_pipelines/v2/setup/aws_eks.yaml

```diff
@@ -0,0 +1,43 @@
+datadog:
+  site: "datadoghq.com"
```

Comment:
The site URL is going to depend on which Datadog site the user is in: https://docs.datadoghq.com/getting_started/site/
Maybe we can say something like:

> The site parameter is based on the Datadog site you are using. For example, the site URL for US1 is `datadoghq.com`. See Datadog Site for more information.